google-cloud-bigquery/google-cloud-bigquery.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.0.0"
 
   gem.add_dependency "google-cloud-core", "~> 1.0"
-  gem.add_dependency "google-api-client", "~> 0.14.0"
+  gem.add_dependency "google-api-client", "~> 0.16.0"
```

Comment:
Are Bigquery Credentials changing to inherit from `Google::Auth::Credentials`? If so, then a dependency on `googleauth ~> 0.6.1` is needed here as well.

Reply:
Not until https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1729 is done. Even after that issue is resolved, I'd expect the dependency to be pull in transitively via `google-cloud-core` rather than in the individual gemspecs for the gems with manual layers.